pymodbus/diag_message.py

```diff
@@ -3,6 +3,46 @@
 These need to be tied into a the current server context
 or linked to the appropriate data
 """
+
+__all__ = [
+    "DiagnosticStatusRequest",
+    "DiagnosticStatusResponse",
+    "ReturnQueryDataRequest",
+    "ReturnQueryDataResponse",
+    "RestartCommunicationsOptionRequest",
+    "RestartCommunicationsOptionResponse",
+    "ReturnDiagnosticRegisterRequest",
+    "ReturnDiagnosticRegisterResponse",
+    "ChangeAsciiInputDelimiterRequest",
+    "ChangeAsciiInputDelimiterResponse",
+    "ForceListenOnlyModeRequest",
+    "ForceListenOnlyModeResponse",
+    "ClearCountersRequest",
+    "ClearCountersResponse",
+    "ReturnBusMessageCountRequest",
+    "ReturnBusMessageCountResponse",
+    "ReturnBusCommunicationErrorCountRequest",
```

Comment:
please format

Reply:
What do you mean by "format" here?  It's the same formatting as it was before

Reply:
my GitHub showed that not all entries was aligned.

Reply:
Okay I believe this is good as it's aligned.  Resolving this one